crates/rolldown/src/stages/generate_stage/advanced_chunks.rs

```diff
@@ -357,11 +357,11 @@ fn add_module_and_dependencies_to_group_recursively(
     return;
   }
 
-  let Module::Normal(module) = &module_table[module_idx] else {
+  let Module::Normal(_) = &module_table[module_idx] else {
```

Comment:
```suggestion
  if !module_table[module_idx].is_normal() {
```
(optional) Maybe this is more simple?